instance/editions.go

```diff
@@ -38,8 +38,21 @@ func (s *Store) confirmEdition(ctx context.Context, datasetID, edition, instance
 			editionDoc = models.CreateEdition(s.Host, datasetID, edition)
 			log.Debug("created new edition", logData)
 		} else {
-			log.Debug("edition found, updating", logData)
+
 			action = UpdateEditionAction
+
+			if editionDoc.Current == nil && editionDoc.Next == nil {
+				log.InfoCtx(ctx, "edition update document has no links. Aborting edition update.", logData)
```

Comment:
no periods on end of log messages (throughout, ill stop repeating myself now 😂 )